ui-components/src/locales/general.json

```diff
@@ -1235,6 +1235,206 @@
     "WI": "Wisconsin",
     "WY": "Wyoming"
   },
+  "neighborhoods":{
+    "Midtown": "Midtown",
```

Comment:
Same comment here, could you alphabetize?

Reply:
Done